src/Modal/index.js

```diff
@@ -250,6 +250,9 @@ const ModalWrapperContent = styled.div`
         // or fixed when fixedFooter (overwrite -ms-page)
         position: ${({ fullyScrolled, fixedFooter }) =>
           (fullyScrolled && fixedFooter && "static") || (fixedFooter && "fixed")};
+        // for IE there's need to be added inset box-shadow with same background as footer has
```

Comment:
wtf

Reply:
😅 IE is just adding some transparent 1px line, due to `border-radius`. This is probably the only solution